server/src/server-settings.js

```diff
@@ -21,7 +21,9 @@ module.exports = {
   "navClient": {
     "username": secrets.navClient.username,
     "password": secrets.navClient.password,
-    "port": 44444,
+    "walletPassphrase": secrets.navClient.walletPassphrase,
+    "walletUnlockTime": 100000,

```

Comment:
we need to tone this down at some point, unlocking for 100,000 seconds is slightly long 😛